### PR TITLE
Harry background

### PR DESCRIPTION
**Pull request by nickine**
#### What does this PR do? (please provide any background)
a couple of things
1) sorts out the backgrounds
2) sorts out the breadcrumbs - no progress trackers for Harry
3) Gets grey tones added into Harry Potter 
4) Replaces Grey with Gray comments since that's a thing

#### How can this be tested?
`cd pattern-library && npm link ../brands/harry-potter`
I can also show you on my machine

#### Any tech debt?
No

#### Screenshots / Screencast
screen shot 2017-02-22 at 14 31 09

#### What gif best describes how you feel about this work?
yeahhh

- [X] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: brands/airparks/src/less/colours.less [hunk 1]
```diff
@@ -18,7 +18,7 @@
 @color-purple:                 #a52752;
 @color-purple-light:           #f5e2e7;
 
-// Greys
+// Grays
```
Comment (jodiedoubleday) [inline brands/airparks/src/less/colours.less]: 😱

File: brands/harry-potter/src/less/backgrounds.less [hunk 2]
```diff
@@ -4,11 +4,46 @@
...
+      background-color: @bg-muted-inverse-normal;
+    }
+  }
+  .bg-default.bg-muted,
+  .bg-default .bg-muted {
+    background-color: @bg-muted;
+  }
+}
+// Inverse backgrounds
+.bg-tertiary {
```
Comment (jodiedoubleday) [inline brands/harry-potter/src/less/backgrounds.less]: This is under the inverse heading/commenting but does not us inverse mixin?

File: brands/harry-potter/src/less/breadcrumbs.less [hunk 3]
```diff
@@ -1 +1,10 @@
 @import '../../../brand-globals/src/less/breadcrumbs';
+
+.breadcrumb {
+  text-transform: capitalize;
+  color: @gray-lighter;
+  margin-bottom: 0;
+ & >li+li:before {
+    padding-left: 10px;
```
Comment (jodiedoubleday) [inline brands/harry-potter/src/less/breadcrumbs.less]: is there a variable for this? `@padding-base-horizontal`

File: brands/airparks/src/less/colours.less (hunk 1 above)
Comment (jodiedoubleday) [inline brands/airparks/src/less/colours.less]: 😱

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: brands/airparks/src/less/colours.less (hunk 1 above)
Comment (nickine) [inline brands/airparks/src/less/colours.less]: you heard

**Pull request merged**